pump/storage/log.go

```diff
@@ -369,7 +369,7 @@ func seekToNextRecord(reader *bufio.Reader) (bytes int, err error) {
 			return
 		}
 
-		reader.Discard(1)
+		reader.Discard(1) //nolint: errcheck
```

Comment:
Add comment for why it will never happen.
and just panic or `var _ = reader.Discard(1)` ?

Reply:
Fixed.